tasks/src/text-to-image/about.md

```diff
@@ -19,3 +19,5 @@ Architects can utilise the models to construct an environment based out on the r
 ## Useful Resources
 - [MinImagen - Build Your Own Imagen Text-to-Image Model](https://www.assemblyai.com/blog/minimagen-build-your-own-imagen-text-to-image-model/)
 - [OpenAI Blog - Dall E](https://openai.com/blog/dall-e/)
+
+This page was made possible thanks to efforts of [Ishan Dutta](https://huggingface.co/ishandutta) and [Oğuz Akif](https://huggingface.co/oguzakif).
```

Comment:
Could we add a demo for this one? Should we straightforward as we support both images and text

Reply:
sure let me do this

Reply:
![Ekran Resmi 2022-11-14 14 26 03](https://user-images.githubusercontent.com/53175384/201671625-c0c1aa0b-841c-410e-8c36-ac0b2d7eb32c.png)
Does this look good?

Reply:
Yes it looks amazing!